go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/hashicorp/vault/api v1.13.0
 	github.com/hashicorp/vault/api/auth/approle v0.6.0
 	github.com/itchyny/gojq v0.12.12
-	github.com/jackc/pgconn v1.14.3
+	github.com/jackc/pgconn v1.14.3 // indirect
```

Comment:
This is now indirect, should be moved

Reply:
the rule to sort direct vs indirect reports is not enforcable, unless we have some tooling that would actually sort imports for us.. we can't expect devs to sort imports if `go mod tidy` does not do it for us.